src/git/extension.ts

```diff
@@ -29,12 +33,19 @@ type ActionableMessageItem = MessageItem & {
     action: () => void;
 }
 
+
 export class Extension {
     private readonly disposable: Disposable;
     private readonly blame: Blamer;
     private readonly view: StatusBarView;
     private readonly headWatcher: HeadWatch;
 
+    /**
+     * decorationCharPosition set to 1024 because normally lines are not that big
+     */
+    private readonly decorationCharPosition: number = 1024;
+    private decorationType: TextEditorDecorationType = window.createTextEditorDecorationType({});
```

Comment:
Please move the initialization of the `decorationType` to the constructor to keep in more in line with the other private variables.

Reply:
Also please add it to the `dispose` function below.